frontend/src/styling/themeHomepage.js

```diff
@@ -0,0 +1,17 @@
+import {createTheme} from "@mui/material";
+
+const themeHomepage = createTheme({
+
+    palette: {
+        type: 'dark',
+        primary: {
+            main: '#3f51b5',
+
+        },
+        secondary: {
+            main: '#f50057',
+        },
+    },
+});
+
```

Comment:
Yor are creating several themes for several components. Build one theme for your app. Then you can edit colors and general stuff in one place. Thats what the theme are for

Reply:
redundant, don't use them so remains only one theme. thanks for the hint. removed all others except theme.